api/src/main/scala/io/guanaco/alerta/api/Alert.scala

```diff
@@ -12,7 +12,8 @@ case class Alert(
     text: Option[String] = None,
     value: Option[String] = None,
     correlate: Option[Seq[String]] = None,
-    timeout: Long = 7 * 24 * 60 * 60 /* 7 days */,
+    //timeout: Long = 7 * 24 * 60 * 60 /* 7 days */,
+    timeout: Option[Long] = Some(7 * 24 * 60 * 60) /* 7 days */,
```

Comment:
Ik denk dat je hier met `None` zou moeten beginnen. Een alert heeft standaard geen timeout, je kan die toevoegen in de `AlertaConfig` of anders een default invullen vanuit de config.